electrum/gui/qt/three_keys_dialogs.py

```diff
@@ -77,15 +86,15 @@ def __init__(self, *args, **kwargs):
 
 
 class InsertPubKeyDialog(QVBoxLayout):
-    def __init__(self, parent, message_label):
+    def __init__(self, parent, message_label, disallowed_keys: List[str] = []):
```

Comment:
Initialization of default mutable parameter like `disallowed_keys: List[str] = []` can cause unpredictable behavior in future usage of this class, see https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument. Safer solution (like from js):
```
def __init__(self, parent, message_label, disallowed_keys: List[str] = None):
    disallowed_keys = disallowed_keys or []
```

Reply:
Done